nox/virtualenv.py

```diff
@@ -39,6 +39,23 @@
 _SYSTEM = platform.system()
 
 
+def find_uv() -> tuple[bool, str]:
+    uv = shutil.which("uv")
+    if uv is not None:
+        return True, uv
+
+    # Look for uv in Nox's environment as well, to handle `pipx install nox[uv]`.
+    with contextlib.suppress(ImportError, FileNotFoundError):
+        from uv import find_uv_bin
+
+        return True, find_uv_bin()
+
+    return False, "uv"
+
+
+HAS_UV, UV = find_uv()
```

Comment:
We could use a single variable, `UV`, and set it to None if UV wasn't found.

Reply:
I had this in an earlier version, but it's awkward because you have to use `UV or "uv"` in the subprocess calls.

Reply:
Ahh, so UV is always a string, and set to "uv" if not found? Okay. I though it was already None in that case.